rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java

```diff
@@ -174,15 +238,16 @@ JobId submit(File job, Map<String, String> variables)
      * precious result.
      * <p>
      *
-     * @param job       a job provided as a url
+     * @param job a job provided as a local File
      * @param variables job variables to use during the job execution
+     * @param genericInfos map with generic informations
      * @return the generated new job ID.
      * @throws NotConnectedException
      * @throws JobCreationException
      * @throws PermissionException
      * @throws SubmissionClosedException
      */
-    JobId submit(URL job, Map<String, String> variables)
+    JobId submit(File job, Map<String, String> variables, Map<String, String> genericInfos)
             throws NotConnectedException, PermissionException, SubmissionClosedException, JobCreationException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)